components/dashboard/src/components/podkit/combobox/Combobox.tsx

```diff
@@ -84,11 +84,14 @@ export const Combobox: FunctionComponent<ComboboxProps> = ({
 
     const setActiveElement = useCallback(
         (element: string) => {
+            if (!filteredOptions.find((el) => element === el.id)?.isSelectable) {
```

Comment:
This was a weird bug in which you could select the Bitbucket server help message as a context URL by clicking on it with your pointer and pressing <kbd>Enter</kbd>